src/memory-analyzer/analyze_symbol.h

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018 Author: Malte Mues
+#ifdef __linux__
```

Comment:
FWIW instead of these ifdefs all over the place we should really just exclude the entire memory analyzer from the build when we're not building on linux (+ maybe OSX? Does this work on OSX?)